addons/web_editor/static/src/scss/web_editor.frontend.scss

```diff
@@ -35,11 +35,13 @@
 // GRID LAYOUT
```

Comment:
Commit message:
- > on top of each others.

-> each other
- > will be set depending of

 -> depending on

- > Point of attention when trying to replicate the issue:
When you enable grid mode, the `data-row-count=` attribute on the grid
element will be set depending of the available size.
So be sure to not enable the grid mode on a megamenu which is already
hidden in the extra items when you are in edit mode, as that attribute
would be something like `23` instead of something like `6` when doing it
normally. And in such a case, there is no bug.

The explanation is not really correct here because it is not `data-row-count` nor the available size that control that. The real explanation is that when toggling the grid mode, the columns are placed as closely as possible to how they were placed in normal mode, so if they are placed on top of each other, they will be placed the same way in grid mode, which will result in a "vertical" grid (and thus the `data-row-count` (which really is a counter) will be set to a higher but correct value, since it counts the produced rows) and the bug will not happen because since they are already like how it is supposed to look in this case, they will stay the same way :slightly_smiling_face: 

- > This commit simply disable 

-> disables
- > it will thus behaves

-> behave

- > it will thus behaves as if it was not in grid mode or if it was in grid mode but in
mobile size where the grid mode is disabled despite being set on the
element.

You can simply say "it will thus behave like the grid mode in mobile view and therefore be responsive.", I think it is clear enough to understand what it means :wink:

Reply:
I changed a bit the message, should be good now